examples/github-notification-filters/app.tsx

```diff
@@ -0,0 +1,212 @@
+/** @jsx h */
+import { autocomplete } from '@algolia/autocomplete-js';
+import { createTagsPlugin, Tag } from '@algolia/autocomplete-plugin-tags';
+import { h, render } from 'preact';
+import qs from 'qs';
+
+import {
+  FilterHeader,
+  PanelLayout,
+  PostfixItem,
+  PrefixItem,
+  QueryItem,
+  TagItem,
+} from './components';
+import { items } from './items';
+import { AutocompleteItem, NotificationFilter } from './types';
+import { getAlltags, groupBy, splitQuery } from './utils';
+
+import '@algolia/autocomplete-theme-classic';
+
+const queryParameters = qs.parse(location.search, { ignoreQueryPrefix: true });
+
+const tagsPlugin = createTagsPlugin<AutocompleteItem, NotificationFilter>({
+  initialTags: Object.entries(queryParameters).flatMap(([token, values]) =>
```

Comment:
This can be its own variable to facilitate reading?

Reply:
Yes, addressed in 3e86422.